### DISCUSSION TITLE
Functionalities

### DISCUSSION THREAD

**Discussion by Imeneallouche** [not1cyyy/SSRFHunter#3] [Q&A] [answered]
What are the functionalities of this SSRF hunter?

**Comment by not1cyyy** [comment, marked as answer]
This is a tool that tests for potential SSRF vulnerabilities in a provided website. as for the current release, we're still trying to improve the tool to do more than that.

You can use the tool by running this command `python3 main.py -u {TARGET_URL}` and it will automatically scan it using some known vulnerable URL parameters.
